scripts.d/support/install-setup-ng

```diff
@@ -17,34 +17,41 @@ function check-writable()
 function template-conversion()
 {
     local PREFIX=$1
+    local DISTRIBUTION_ID=$2
+    local OS_VERSION=$3
     # Convert the template files (omniidl, h5hlcc, ...)
     #
 
     local ORIG_BINDIR="__BINDIR__"
     local BINDIR=$(basename $(ls -1d $PREFIX/BINARIES-*))
 
-    for SCRIPTS in omniORB/bin/omniidl \
-                       hdf5/bin/h5hlcc hdf5/bin/h5cc \
-                       netcdf/lib/pkgconfig/netcdf.pc \
-                       PyQt/bin/pyuic5;
+    local FILES_LIST
+    if [[ $DISTRIBUTION_ID -eq "Debian" && $OS_VERSION -eq "9" ]]; then
+        FILES_LIST=(omniORB/bin/omniidl hdf5/bin/h5hlcc hdf5/bin/h5cc netcdf/lib/pkgconfig/netcdf.pc PyQt/bin/pyuic5)
+    else
+        FILES_LIST=(omniORB/bin/omniidl hdf5/bin/h5hlcc hdf5/bin/h5cc)
+    fi
+    for SCRIPTS in ${FILES_LIST[*]} ;
     do
         sed -i "s,${ORIG_BINDIR},$PREFIX/${BINDIR},g" $PREFIX/$BINDIR/$SCRIPTS
     done
 
-    #  For sip
+    if [[ $DISTRIBUTION_ID -eq "Debian" && $OS_VERSION -eq "9" ]]; then
```

Comment:
likewise:
please use: [[ $DISTRIBUTION_ID -eq "Debian" ]] && [[ $OS_VERSION -eq "9" ]]; then

(and there is another ref in the file to be changed I think).